example/spec/helpers/testUtils.js

```diff
@@ -117,10 +127,18 @@ const loadConfig = async (type = 'app') => {
     configFromFile.bucket,
     `${configFromFile.stackName}/workflows/buckets.json`
   );
+  const buckets = JSON.parse(bucketsObject.Body.toString());
+
+  // assume only one internal bucket in config.
+  if (isNil(configFromFile.bucket)) {
+    configFromFile.bucket = buckets.filter(
```

Comment:
1. `buckets.first()` would be a better function to use here
2. This is going to set `config.bucket` to an object, but it should be a string.

Reply:
Removed.